patches/0006-Add-goexperiment.systemcrypto-alias.patch

```diff
@@ -0,0 +1,271 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
This patch should go before the `Add-crypto-backend-foundation.patch`, having `goexperiment.systemcrypto` will simplify patching build tag headers down the line.

Both two could even be merged, if it makes sense to you.

Reply:
👍 

I think it's focused enough on `go build` that it makes sense to keep separate. It seems pretty independent from the foundation patch, too. (I don't see a reason the foundation patch would use the systemcrypto tag.)